src/current/v25.2/alter-role.md

```diff
@@ -59,73 +64,96 @@ The following statement prevents the user from using password authentication and
 
 ~~~ sql
-root@:26257/defaultdb> ALTER ROLE carl WITH CREATEROLE CREATELOGIN;
+ALTER ROLE carl WITH CREATEROLE CREATELOGIN;
 ~~~
 
 ### Allow a role to create and rename databases
 
 The following example allows the role to [create]({% link {{ page.version.version }}/create-database.md %}) or [rename]({% link {{ page.version.version }}/alter-database.md %}#rename-to) databases:
 
 ~~~ sql
-root@:26257/defaultdb> ALTER ROLE carl WITH CREATEDB;
+ALTER ROLE carl WITH CREATEDB;
 ~~~
 
 ### Allow a role to pause, resume, and cancel non-admin jobs
 
 The following example allows the role to [pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), and [cancel]({% link {{ page.version.version }}/cancel-job.md %}) jobs:
 
 ~~~ sql
-root@:26257/defaultdb> ALTER ROLE carl WITH CONTROLJOB;
+ALTER ROLE carl WITH CONTROLJOB;
 ~~~
 
 ### Allow a role to see and cancel non-admin queries and sessions
 
 The following example allows the role to cancel [queries]({% link {{ page.version.version }}/cancel-query.md %}) and [sessions]({% link {{ page.version.version }}/cancel-session.md %}) for other non-`admin` roles:
 
 ~~~ sql
-root@:26257/defaultdb> ALTER ROLE carl WITH CANCELQUERY VIEWACTIVITY;
+ALTER ROLE carl WITH CANCELQUERY VIEWACTIVITY;
 ~~~
 
 ### Allow a role to control changefeeds
 
 The following example allows the role to run [`CREATE CHANGEFEED`]({% link {{ page.version.version }}/create-changefeed.md %}):
 
 ~~~ sql
-root@:26257/defaultdb> ALTER ROLE carl WITH CONTROLCHANGEFEED;
+ALTER ROLE carl WITH CONTROLCHANGEFEED;
 ~~~
 
 ### Allow a role to modify cluster settings
 
 The following example allows the role to modify [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}):
 
 ~~~ sql
-root@:26257/defaultdb> ALTER ROLE carl WITH MODIFYCLUSTERSETTING;
+ALTER ROLE carl WITH MODIFYCLUSTERSETTING;
+~~~
+
+### Allow a role to bypass row-level security (RLS)
+
+To allow a [role]({% link {{ page.version.version }}/security-reference/authorization.md %}#users-and-roles) to bypass [row-level security]({% link {{ page.version.version }}/row-level-security.md %}), execute the following statement to grant the [`BYPASSRLS`]({% link {{ page.version.version }}/security-reference/authorization.md %}#bypassrls) privilege:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER ROLE carl WITH BYPASSRLS;
+~~~
+
+To disable the role's ability to bypass RLS, execute the following statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER ROLE carl WITH NOBYPASSRLS;
+~~~
+
+To see all of the roles that can bypass RLS, execute the following query as an [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role):
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT rolname FROM pg_roles WHERE rolbypassrls = true
```

Comment:
Similar comment here about bug https://github.com/cockroachdb/cockroach/issues/146228.

Reply:
same q re: should we remove for now?